src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -82,4 +100,36 @@ public Command parseCommand(String userInput) throws ParseException {
         }
     }
 
+    /**
+     * Checks if command is a substring of a full command.
+     *
+     * @param command user input command string
+     * @return  full command if input is a substring, else return the original input command
+     * @throws ParseException if the input matches to multiple commands
+     */
+    private String fullCommand(String command) throws ParseException {
+        final String initialValue = "";
+        String possibleCommand = initialValue;
+
+        for (String commandWord : COMMANDS) {
+            // Input command matches the full command.
+            if (commandWord.startsWith(command)) {
+                // Input command matches with multiple full commands.
+                if (!possibleCommand.equals(initialValue)) {
+                    throw new ParseException(MESSAGE_UNCLEAR_COMMAND);
```

Comment:
We aim to get 100% diff coverage from now on

Reply:
No harm adding this check in case there are future commands with similar prefixes.

Reply:
I say we aim to get 100% diff coverage means you should add test for this feature.

Reply:
Oh mb I misunderstood your statement. I think it's possible to test that part if we parse "e" as command since it'll map to edit and exit commands. Though aside from this specific case, there's indeed no other way to test it.

Reply:
That would be sufficient then.